docs: fix grammar and typos (#1196)
dequelabs/cauldron: docs/pages/components/Stepper.mdx

<ComponentProps
  children={{

>     description: 'children to be rendered as the step\'s visible label (if a not visible label is desired, use the "tooltip" prop instead)'

Reviewer: ```suggestion
    description: 'children to be rendered as the step\'s visible label (if no visible label is desired, use the "tooltip" prop instead)'
```

I think we probably meant to do this?

Author: I can switch it back if that's the case!
